skyportal/facility_apis/atlas.py

```diff
@@ -0,0 +1,310 @@
+import requests
+import json
+from datetime import datetime, timedelta
+from astropy.time import Time
+import numpy as np
+import pandas as pd
+from io import StringIO
+
+from . import FollowUpAPI
+from . import FakeApplication, FakeRequest
+from baselayer.app.env import load_env
+
+from ..utils import http
+
+env, cfg = load_env()
+
+if cfg['app.atlas.port'] is None:
+    ATLAS_URL = f"{cfg['app.atlas.protocol']}://{cfg['app.atlas.host']}"
+else:
+    ATLAS_URL = (
```

Comment:
I see we've got it this way in a couple of places in the config.  A bit silly really.  This could just be `ATLAS_URL=https://foo.bar:3125`

Reply:
I think some earlier telescope API reviews requested splitting them up.